scripts/deploy/__init__.py

```diff
@@ -177,3 +180,28 @@ def run_deploy_twocrypto(chain: str, fee_receiver: str) -> None:
 @click.argument("coins", type=click.STRING)
 def run_deploy_twocrypto(chain: str, name: str, symbol: str, coins: str) -> None:
     deploy_pool(chain, name, symbol, coins.split(","))
+
+
+@deploy_commands.command("test_pools", short_help="deploy test tokens and pool on devnet")
+@click.argument("chain", type=click.STRING)
+def run_test_pools_deployment(chain: str) -> None:
+    assert "devnet" in chain, "Command only for devnets"
+
+    chain_settings = get_chain_settings(chain)
+    chain_settings.file_name = chain
```

Comment:
here it makes sense to use the is_devnet check as well.

The reason for not relying on the naming is: if we want to integrate this deployment service on RaaS provider's infra, then we are much better off not relying on folder structures and sticking to the contents of the config yaml file